InterfacePython/testBasicIO.py

```diff
@@ -410,6 +410,62 @@ def checkReadWriteMisc(self):
                 gridB.read("testSave")
                 ttc.compareGrids(gridA, gridB)
 
+    def checkReadWriteCustomTabulated(self):
+        '''
+        Test reading and writing of the CustomTabulated class.
+        '''
+        description = "testCT"
+        def create_nodes(j):
+            return np.linspace(-1.0, 1.0, num=j)
+        def create_weights(j):
+            weights = np.linspace(0.0, 1.0, num=j)
+            weights = 2.0 * weights / weights.sum()
+            return weights
+        # Read and write from explicitly given data.
+        for i in range(4):
+            num_levels = i
+            num_nodes = np.array([3*(j+1) for j in range(i)])
+            precision = np.array([2*(j+1)-1 for j in range(i)])
+            nodes = [create_nodes(j) for j in num_nodes]
+            weights = [create_weights(j) for j in num_nodes]
+            ctA = TasmanianSG.makeCustomTabulatedFromData(num_levels, num_nodes, precision, nodes, weights, description)            # .
+            ctB = TasmanianSG.CustomTabulated()
+            ctA.write("testSave")
+            ctB.read("testSave")
+            for i in range(num_levels):
+                read_weights, read_nodes = ctB.getWeightsNodes(i)
+                np.testing.assert_almost_equal(read_weights, weights[i])
```

Comment:
This is failing now.

Are you sure `np.linspace()` does what you expect? Can it really return only one point?